libs/clients/national-registry/v3-applications/src/index.ts

```
@@ -0,0 +1,4 @@
+export { NationalRegistryV3ApplicationsClientModule } from './lib/nationalRegistryV3Applications.module'
+export { NationalRegistryV3ApplicationsClientConfig } from './lib/nationalRegistryV3Applications.config'
+export { NationalRegistryV3ApplicationsClientService as NationalRegistryV3ApplicationsClientService } from './lib/nationalRegistryV3Applications.service'
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)